src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/debtpositiontype/DebtPositionTypeIngestionActivityImpl.java

```diff
@@ -58,7 +58,7 @@ protected DebtPositionTypeIngestionFlowFileResult handleRetrievedFiles(List<Path
           DebtPositionTypeIngestionFlowFileDTO.class, (csvIterator, readerException) ->
               debtPositionTypeProcessingService.processDebtPositionType(csvIterator,
                   readerException,
-                  ingestionFlowFileDTO, workingDirectory));
+                  ingestionFlowFileDTO, workingDirectory), ingestionFlowFileDTO.getFileVersion());
```

Comment:
This flow has not version set on csv DTO (neither ingestionFlowFileDTO.getFileVersion() is probably set)
```suggestion
                  ingestionFlowFileDTO, workingDirectory), null);
```